logistics/scripts/modules/settings/settings-management.psm1

```diff
@@ -0,0 +1,351 @@
+                    Default = "Debug"
+                }
+            }
+        }
+        "EdFi.Ods.Api.IntegrationTestHarness" = @{
+            Urls              = "http://localhost:8765"
+            ApiSettings       = @{
+                Engine = ""
+            }
+            ConnectionStrings = @{ }
+        }
+        "EdFi.Ods.SandboxAdmin.Web"           = @{
+            Urls              = "http://localhost:38928"
+            Engine            = ""
+            ConnectionStrings = @{ }
+            Logging           = @{
+                LogLevel = @{
+                    Default = "Debug"
+                }
+            }
+        }
+        "EdFi.Ods.SwaggerUI"                  = @{
+            Urls             = "http://localhost:56641"
+            WebApiVersionUrl = "http://localhost:54746"
+            Logging          = @{
+                LogLevel = @{
+                    Default = "Debug"
+                }
+            }
+        }
+    }
+}
+
+function Get-CredentialSettingsByProject {
+
+    function Get-RandomString([int] $length = 20) {
+        return ([char[]]([char]65..[char]90) + ([char[]]([char]97..[char]122)) + 0..9 | Sort-Object { Get-Random })[0..$length] -join ''
+    }
+
+    $populatedKey = Get-RandomString
+    $populatedSecret = Get-RandomString
+
+    return @{
+        "EdFi.Ods.SandboxAdmin.Web" = @{
+            User = @{
+                "Test Admin" = @{
+                    Email             = "test@ed-fi.org"
+                    Password          = Get-RandomString
+                    Admin             = "true"
+                    NamespacePrefixes = @(
+                        "uri://ed-fi.org"
+                        "uri://gbisd.org"
+                    )
+                    Sandboxes         = @{
+                        "Populated Demonstration Sandbox" = @{
+                            Key     = $populatedKey
+                            Secret  = $populatedSecret
+                            Type    = "Sample"
+                            Refresh = "true"
+                        }
+                        "Minimal Demonstration Sandbox"   = @{
+                            Key     = Get-RandomString
+                            Secret  = Get-RandomString
+                            Type    = "Minimal"
+                            Refresh = "true"
+                        }
+                    }
+                }
+            }
+        }
+        "EdFi.Ods.SwaggerUI"        = @{
+            SwaggerUIOptions = @{
+                OAuthConfigObject = @{
+                    ClientId     = $populatedKey
+                    ClientSecret = $populatedSecret
+                }
+            }
+        }
+    }
+}
+
+function Get-ConnectionStringKeyByDatabaseTypes {
+    return @{
+        Ods      = 'EdFi_Ods'
+        Admin    = 'EdFi_Admin'
+        Security = 'EdFi_Security'
+        Master   = 'EdFi_Master'
+    }
+}
+
+function Get-ConnectionStringsByEngine {
```

Comment:
Similar here, this seems to be providing the default connection strings for local development.  Documentation and/or function rename to clarify?

Reply:
modified to `Get-DefaultDevelopmentSettingsByEngine`